### PR TITLE
Add SQL Binding support

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Adds Python language support for SQL Input/Output bindings

https://github.com/Azure/azure-functions-sql-extension

(this is a continuation of https://github.com/Azure/azure-functions-python-library/pull/118 - that one can be closed)

### CODE REVIEW

**Review by YunchuWang — APPROVED** [review]

**Review by gavin-aguiar — APPROVED** [review]

File: azure/functions/_abc.py [hunk 1]
```diff
@@ -422,3 +422,32 @@ class OrchestrationContext(abc.ABC):
     @abc.abstractmethod
     def body(self) -> str:
         pass
+
+
+class SqlRow(abc.ABC):
```
Comment (vrdmr) [inline azure/functions/_abc.py:427]: @YunchuWang @gavin-aguiar - Is there a ClassWide coverage NoQA to skip the coverage the abstract class?

**Review by vrdmr — APPROVED** [review]
LGTM.

Please add a NoQA flag on the code lines in the abstract class which cannot be covered.

File: azure/functions/_abc.py (hunk 1 above)
Comment (Charles-Gagnon) [reply azure/functions/_abc.py:427]: This should do it right?

`# flake8: noqa`

https://github.com/Azure/azure-functions-python-library/blob/dev/azure/functions/_thirdparty/werkzeug/_compat.py#L1

That seems like something that should be done separately though. Or is this a blocking issue for getting this PR in?

Comment (vrdmr) [reply azure/functions/_abc.py:427]: This isn't blocking but would be good to get that change in along with the code it's checking. Lets get this PR in and we can get the NoQA later.

**Pull request merged**